pkgs/by-name/li/livi/package.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
```

Comment:
Should I order the args alphabetically here?
I noticed that some packages put builtins like lib, stdenv,... first and then order the deps alphabetically.

Reply:
Not needed. You can also sort them in the order they are used. It's up to you. But `lib` should be placed in first order.